docs/index.html

```diff
@@ -21,44 +21,44 @@
 					<img src="https://i.imgur.com/vKFMlIj.png" alt="l5 logo" width="50%" style="text-align: none;" />
 				</section>
 
-				<section>
-					<h2>From Zero to Auto Pilot:</h2>
-					<h2>Exploring Kubernetes Operator Capability Levels</h2>
-				</section>
-
 
 				<section>
-					<h3><b>What is an <span style="color:rgb(212, 18, 18);">operator</span> ?</b></h3>
-					<p class="fragment fade-up" style="color:rgba(255, 255, 255, 0.75);">An <u>opertator</u> is a method of packaging, deploying, and managing an application by extending the functionality of the Kubernetes API </p>
+					<h2>From Zero to <span style="color:red">Auto Pilot</span>:</h2>
+					<p>Exploring Kubernetes Operator Capability Levels</p>
 				</section>
 
+
 				<section data-auto-animate>
-					<h3>What is the L5 operator &</h3>
+					<h3>What is the L5 operator <a href="https://bestie-rescue.herokuapp.com/">&</a></h3>
 					<h3>Why was it created?</h3>
 					<aside class="notes">
-						- The L5 operator is a demo operator that manages and consumes a flask app with a post gres database while utilizes
+						- The L5 operator is an operator that manages and consumes a flask app with a postgres database utilizing
```

Comment:
Something like we will showcase the capabilities using an operator that manages a flask app that uses a postgres database..